tools/rostest/src/rostest/rostest_main.py

```diff
@@ -98,6 +98,10 @@ def rostestmain():
     parser.add_option("--results-base-dir", metavar="RESULTS_BASE_DIR",
                       help="The base directory of the test results. The test result file is " +
                            "created in a subfolder name PKG_DIR.")
+    parser.add_option("-r", "--reuse-master", action="store_true",
+                      help="Connect to an existing ROS master instead of spawning a new ROS master on a custom port")
+    parser.add_option("-c", "--clear", action="store_true",
+                      help="Clear all parameters when connecting to existing ROS master (useful with --reuse-master)")
```

Comment:
Please clarify that this option is only effective when `--reuse-master` is specified. argparse could even error out when the option is used without `--reuse-master`.